pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -149,6 +150,8 @@ def read_mwa_corr_fits(
         self,
         filelist,
         use_cotter_flags=False,
```

Comment:
I think it would be nice to have this default to `True` if the file is provided in the file list (because then I assume people want it applied). To do this, you could default it to `None` here. Then in lower down you can check if it is `None` and if it is, you can set it to `True` if the file is included in the file list and `False` otherwise.